Assets/Plugins/Editor/Uplift/UpliftManager.cs

```diff
@@ -323,7 +323,7 @@ public void InstallPackages(PackageRepo[] targets, bool updateLockfile = true)
 						if (Upbring.Instance().InstalledPackage.Any(ip => ip.Name == pr.Package.PackageName))
 						{
 							Debug.Log("update " + pr.Package.PackageName);
-							UpdatePackage(pr, updateLockfile: updateLockfile);
+							UpdatePackage(pr, updateDependencies: false, updateLockfile: updateLockfile);
```

Comment:
So this wasn't a regression from #88.

Any chance we can have a unit test for this?

Reply:
Looks like it wasn't.
> Any chance we can have a unit test for this?
On it.

Reply:
Turns out that's going to be super complicated to test. The package wasn't installed in the end, so the unit test would need to make sure that InstallPackage is not called with the problematic version, but we don't have clean separation of concerns which makes it hard to mock or inject.